adding_species_manuscript.tex

```diff
@@ -652,29 +656,13 @@ \subsection*{Filling out the missing pieces}
 \end{table}
 
 For many species it is difficult to obtain estimates of all necessary model parameters.
-Table \ref{tab:param-mod} provide suggestions for ways to deal with missing values of various central model parameters.
-The table also mentions the main discrepancies between the simulated data and real genomic data,
-which can be caused by misspecification of each parameter.
-
-Several researchers who participated in the ``Growing the Zoo'' hackathon wished to add species
```

Comment:
merged the discussion in this paragraph into the 'chromosome-level genome assembly' paragraph in the prev section